Dockerfile

```
@@ -0,0 +1,38 @@
+
+FROM openjdk:17-jdk-slim AS builder
+RUN adduser --disabled-password --gecos '' appuser
+
+# 권장: non-root 실행
+USER appuser
+
+# --- 런타임 필수 패키지 설치 --------------------------
+RUN apt-get update && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)